lib/firebaseApi.js

```diff
@@ -0,0 +1,33 @@
+const api = require("./api");
+
+function _list(nextPageToken, projects) {
+  projects = projects || [];
+
+  let path = "/v1beta1/projects?page_size=100";
```

Comment:
Can you take "v1beta1" into a global variable called API_VERSION?

Reply:
Done.